src/add_modern_pagenumbers.py

```diff
@@ -20,6 +20,11 @@
 xml_ns = "{http://www.w3.org/XML/1998/namespace}"
 parser = etree.XMLParser(remove_blank_text=True)
 
+def pad_nr(protocol):
```

Comment:
This is being used in multiple places. Either have it inline, or add to pyriksdagen.

Reply:
@BobBorges @vgkz